client/src/components/question-answer/IndividualAnswer.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react';
+import {sampleAnswersList } from './sampleData.js';
+
+const IndividualAnswer = () => {
+
+  const [ answers, setAnswers ] = useState(sampleAnswersList.results[0].body);
```

Comment:
If you've ever felt you're writing too many useState functions, you can look into this: https://blog.bitsrc.io/best-practices-with-react-hooks-69d7e4af69a7 #4 (useState can be used exactly like the class component’s state. Not just for a single value) to combine your related state into one state object.

Reply:
Thanks! I definitely feel like I am. I will read into this